docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md

```diff
@@ -1 +1 @@
-Ubuntu specific instructions
\ No newline at end of file
+Ubuntu specific instructions
```

Comment:
why is this here/being changed?

Reply:
I updated this document wiki when this was sent to `master`. On `23.0-release`, the contents are different so I reverted those changes in the rebase. I guess there was a space left over when I cleared the conflicting lines.